scripts/calendar/germany_school_holidays_generator.py

```
@@ -0,0 +1,438 @@
+    if unknown_columns:
+        raise ValueError(f"Unsupported KMK holiday columns in {path.name}: {unknown_columns}.")
+
+    rows = {}
+    for raw_row in table[1:]:
+        raw_label = raw_row[0] or ""
+        state_label = _normalize_state_label(raw_label)
+        subdiv = STATE_LABEL_TO_CODE.get(state_label)
+        if subdiv is None:
+            continue
+        rows[subdiv] = {
+            column: _normalize_cell(cell or "") for column, cell in zip(columns, raw_row[1:])
```

Ruff (0.15.9)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)